app.py

```diff
@@ -163,15 +163,15 @@ def create_base_request(file):
         if not line.isnull().all():
             specification = line['Specification']
             requestAsText += str(specification) + "\n"
-        
+
         line_text = str(line['Key']) + ":" + str(line['Question']) + "; \n"
         requestAsText += line_text
     return requestAsText
 
-    
-        
 
- 
+
+
+
 model = GenerativeModel("gemini-1.0-pro-vision-001")
 projectinit=vertexai.init(project="test-application-2-416219", location="northamerica-northeast1")
```

Comment:
file an issue for use dotenv for these values (taking care of adding .env to .gitignore so it doesn't get checked in)

Reply:
I don't think it is necessary because nobody can access it if they don't have my account. When we gonna put a API next week i'm gonna do it but i don't think this is relevant for this pr!

Reply:
@rngadam I try to check for the workflow and i dont know why it is not working maybe it is just temporary

Reply:
![image](https://github.com/ai-cfia/harvester-prototype/assets/73604556/49b2c68e-f63a-4a98-9162-044239d651d3)

Reply:
> I don't think it is necessary because nobody can access it if they don't have my account. When we gonna put a API next week i'm gonna do it but i don't think this is relevant for this pr!

yes, which is why I suggested creating an issue to postpone the work to a future PR

Reply:
@rngadam please accept the pr before leaving i'm gonna wait for the check to be good before merging

Reply:
Ok i'm gonna make a draft object on harvester project